python/dremio-flight/dremio/arguments/parse.py

```diff
@@ -13,116 +13,124 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 """
-
+import yaml
 import argparse
-import sys
 import certifi
 
 
-class KVParser(argparse.Action):
-    def __call__(self, parser, namespace, values, option_string=None):
-        setattr(namespace, self.dest, [])
+def username_required_validator(config: dict):
+    # if config["options"].get("")
```

Comment:
Is this intentionally commented out?

Reply:
Should have cleaned that up - it's an unused function that I was thinking about.